### ISSUE TITLE
Intelligent rotation

### ISSUE THREAD

**Issue by phil164** [NagVis/nagvis#285]
We are using nagvis to monitor hosts/services of multiple companies and have for each company a own map which are in a rotation.
We would like to create one map with just all companies and the overall status of these companies.
If one of the hosts/services of a company go red, we'd like to jump into the specific company dashboard.
So we have a dynamic/intelligent rotation

**Comment by ekrichbaum** [comment]
I think what you are looking for is the actions tab of the map object.  Set the target to be the specific map that you want. Instead of having the object (hostgroup) go to nagios, have it go to the specific map and from there jump to your nagios if you want that information.

**Comment by ekrichbaum** [comment]
hit comment too fast.   or create the objects on your master map as type map of the other maps.

**Comment by phil164** [comment]
thanks for the comment. I should have mentioned, that our rotation is running on one big tv screen. There is no interaction from us with this tv. So nobody will click on a map object if it turns to warning or error. What we want is the action getting triggered automatically when the status of the map object changes to get to a more detailed map.

**Comment by LarsMichelsen** [comment]
Good idea. But may also be a performance issue with a larger number of objects on the maps, since you would always have to process all states of all your maps in the background to get an idea of the problematic states on the maps.

It may also be interesting which map(s) to show in which mode once there are multiple maps showing up problems.

**Comment by phil164** [comment]
Hello,

I had followig in mind:
- We create service groups for each company with all their services.
- We create a main map with a item for each company service group
- This main map refreshes all 5 sec for example
- When one of the company service groups goes to red it changes to the company specific map
- When multiple companies go to red it goes into a rotation between all company maps which are red (like the rotation we already have)
